tianshou/utils/net/common.py

```diff
@@ -480,7 +550,7 @@ def forward(
         self,
         obs: Union[np.ndarray, torch.Tensor],
         state: Any = None,
-        info: Dict[str, Any] = {},
```

Comment:
I believe all network forward method should have `(obs, state, info)` input signature, no need to use kwargs